htdocs/install/step5.php

```diff
@@ -176,17 +176,24 @@
 		    $sql = "SELECT u.rowid, u.pass, u.pass_crypted";
 		    $sql.= " FROM ".MAIN_DB_PREFIX."user as u";
 		    $resql=$db->query($sql);
-		    if ($resql)
-		    {
-		        $numrows=$db->num_rows($resql);
-    			if ($numrows == 0)
-    			{
-    			    // Define default setup for password encryption
-    			    dolibarr_set_const($db, "DATABASE_PWD_ENCRYPTED", "1", 'chaine', 0, '', $conf->entity);
-    			    dolibarr_set_const($db, "MAIN_SECURITY_SALT", dol_print_date(dol_now(), 'dayhourlog'), 'chaine', 0, '', 0);      // All entities
-    			    dolibarr_set_const($db, "MAIN_SECURITY_HASH_ALGO", 'sha1md5', 'chaine', 0, '', 0);                               // All entities
-    			}
-		    }
+			if ($resql)
+			{
+				$numrows=$db->num_rows($resql);
+				if ($numrows == 0)
+				{
+					$salt = dol_print_date(dol_now(), 'dayhourlog');
+
+					// Define default setup for password encryption
+					dolibarr_set_const($db, "DATABASE_PWD_ENCRYPTED", "1", 'chaine', 0, '', $conf->entity);
+					dolibarr_set_const($db, "MAIN_SECURITY_SALT", $salt, 'chaine', 0, '', 0);      // All entities
+					dolibarr_set_const($db, "MAIN_SECURITY_HASH_ALGO", 'sha1md5', 'chaine', 0, '', 0);                               // All entities
+
+					// Update of $conf->global
+					$conf->global->DATABASE_PWD_ENCRYPTED = 1;
```

Comment:
This looks strange because the dolibarr_set_const already contains the set of $conf->global

Reply:
@eldy 
I replaced everything with a log